minimal-conda-requirements.txt

```diff
@@ -0,0 +1,27 @@
+# Use this file to create a conda environment using:
```

Comment:
conda doesn't (currently) let you use multiple `--file` arguments so I've stuck with the existing two-file structure.